index.html

```diff
@@ -180,7 +180,22 @@ <h3 class="canopy">Frameworks</h3>
                 <h3 class="canopy">Skills</h3>
                 <a href="#"><img class="icons" src="images/right-arrow.png" alt="arrow"></a>
             </div>
+            <section class="contact-section">
+                <h1 id="contact-card-title">Contact me</h1>
+                <p id="contact-paragraph">If you have an application you are interested in developing, a feature that
+                    you
+                    need built or a project that needs coding. I’d love to help with it</p>
+                <form action="https://formspree.io/f/mnqlovzq" method="POST" class="contact-form">
+                    <input type="text" name="_name" placeholder="Name" class="form-field" required>
+                    <input type="email" name="_replyto" placeholder="Email" class="form-field" required>
+                    <textarea name="_message" placeholder="Write your message here" maxlength="500"
+                        class="form-field contact-message"></textarea>
```

Comment:
- [ ] It will be nice If we can't submit an empty message. so please make it require also to prevent such thing.